popup.js

```diff
@@ -169,15 +174,44 @@ function loginProcess() {
         checkAccount();
         chrome.tabs.getSelected(null, function (tab) {
             tablink = tab.url;
-            if (startsWithHTTP(tablink,"http://findagrave.com") || startsWithHTTP(tablink,"http://forums.findagrave.com")) {
-                tablink = tablink.replace("http://findagrave.com", "http://www.findagrave.com");
-                tablink = tablink.replace("http://forums.findagrave.com", "http://www.findagrave.com");
+
+            console.log(collections);
+            // Select collection
+            for (var i=0; i<collections.length; i++) {
+              if (startsWithHTTP(tablink, collections[i].url)) {
```

Comment:
Some collections work with multiple URLs, so we might want to use a matching function instead of `startsWithHTTP` eventually.